Detectors/MUON/MCH/Simulation/include/MCHSimulation/Response.h

```diff
@@ -74,6 +90,10 @@ class Response
 
   //anode-cathode Pitch in 1/cm
   float mInversePitch = 0.0;
+
```

Comment:
If I understand correctly the big picture here, the intent is to have a mode of running where the O2 simulation behaves like AliRoot, for comparison's sake ? 
If that's indeed the case, I guess it would be beneficial to have methods where the constants that are different in both cases are set. 
Concerning the adc : it is indeed 12 bits for Run2, and the individual samples are 10 bits for Sampa chip, but in charge sum mode, that's 20 bits I believe.

Reply:
Yes correct. 
I haven't yet started to reproduce O2, with 20 bits etc. pp, since I haven't yet fully reproduced aliroot. I am currently trying to debug it from the intermediate steps from aliroot.